v2v/tests/src/libnbd/libnbd.py

```
@@ -95,6 +100,91 @@ def test_unsanitized_hostname():
+        3. Verify source disk usage is minimal (~2M) using 'du'.
+        4. Run 'nbdcopy' with --allocated and --synchronous flags from
+        source to destination.
+        5. Verify destination disk usage is exactly 100M.
+        6. Verify destination and source logical sizes are identical.
+        """
+        # Configuration
+        SRC_IMG = os.path.join(data_dir.get_tmp_dir(), "src.img")
+        DEST_ALLOC = os.path.join(data_dir.get_tmp_dir(), "dest_alloc.img")
+
+        # 1. Create sparse source image (100M)
+        process.run(f"truncate -s 100M {SRC_IMG}", shell=True, ignore_status=True)
```

Ruff (0.15.9)

[error] Function call with `shell=True` parameter identified, security issue

(S604)